kernel/arch/x86_64/boot/idt_dispatch.c

```
@@ -0,0 +1,25 @@
+#include "idt.h"
+#include "drivers.h"
+#include <stdint.h>
+
+static void (*s_handlers[256])(void);
+
+int x86_idt_register_handler(int vector, void (*handler)(void)) {
+    if (vector < 0 || vector >= 256 || !handler)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)